src/main/java/edu/kis/powp/jobs2d/command/file/IImportCommand.java

```diff
@@ -0,0 +1,10 @@
+package edu.kis.powp.jobs2d.command.file;
+
+import edu.kis.powp.jobs2d.command.DriverCommand;
+
+import java.util.List;
+
+public interface IImportCommand {
+
+	List<DriverCommand> importCommandSequence(String filename);
```

Comment:
importing should be done from a text, not from a file. Reading text from a file is a separate task (responsibility)